src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+.search {
+  width: 100%;
+  font-family: inherit;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+}
+
+.search__big {
+  height: 70px;
+  line-height: 70px;
+  font-size: 16px;
+  background: url(images/Search.svg) no-repeat 26px 25px/19px;
+  padding-left: 62px;
+}
+
+.search__small {
+  font-size: 14px;
+  height: 42px;
+  background: url(images/Search.svg) no-repeat 13px 15px/11px;
+  padding-left: 33px;
+}
+
+.search__big::placeholder {
+  color: #3d4e61;
+}
+
+.search__small::placeholder {
+  color: #3d4e61;
+}
+
+.search__big:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search__big:focus {
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%)
+    url(images/Search.svg) no-repeat 26px 25px/19px;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  outline: none;
+}
+
+.search__small:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
```

Comment:
repeated rules. We will apply them to .loop:hover
```suggestion

```

Reply:
"loop" - is this another name for my class, which combines the rules or is it an attribute that I haven't learned yet

Reply:
it is just a variant for class name for your image